src/redux/services/helm.ts

```diff
@@ -169,6 +173,7 @@ export function createHelmChart(fileEntry: FileEntry, absolutePath: string, helm
     filePath: fileEntry.filePath,
     name: getHelmChartName(absolutePath),
     valueFileIds: [],
+    otherFilePaths: [],
```

Comment:
should we have a dedicated "templateFilePaths" or what "other" file types does this cover?

Reply:
templateFilePaths sounds better that other